src/actions/forecast.actions.js

```diff
@@ -0,0 +1,20 @@
+import { forecastConstants } from '../constants';
+import { forecastPeriodService } from '../services';
+
+
+export const forecastActions = {
+	fetchForecast,
+};
+
+function fetchForecast(){
+    return async (dispatch,currentState)=>{
+        const {lat, lng} = currentState().city.latLng;
+        try{
+            const forecast = await forecastPeriodService.getForecastByLatLng(lat, lng);
+            dispatch({ type: forecastConstants.GET_BY_LATLNG, forecast:forecast });
```

Comment:
You can use shot form for `forecast`: `{ type: forecastConstants.GET_BY_LATLNG, forecast}`

Reply:
+